### ISSUE TITLE
Add shell args

### ISSUE THREAD

**Issue by 4a-42** [saurabhdaware/vscode-terminal-manager#3]
for example, this is the setup i have for the integrated shell:

    "terminal.integrated.shell.windows": "C:\\SWTools\\cygwin\\bin\\bash.exe",
    "terminal.integrated.shellArgs.windows": ["/bin/xhere", "/bin/bash"],

There doesn't seem to be an easy way to translate this to the extension.

**Comment by saurabhdaware** [comment]
Hey I totally missed out shellArgs thanks for reminding! Yeah we can totally do that so in settings file we can add new key called shellArgs:[] and then pass it from vscode.window.createTerminal()

so new terminals.json would look like 
```json
[
    {
        "label":"Windows",
        "shellPath":"C://Windows//System32//cmd.exe",
        "shellArgs": ["/k" ,"Hello cmd user"]
    },
    {
        "label":"Ubuntu",
        "shellPath":"C://Windows//System32//bash.exe",
        "shellArgs": ["/k" ,"Hello ubuntu user"]
    }
]
```

I am keeping this issue open let me know if you want to work on this one

**Comment by 4a-42** [comment]
I've never made a vscode extension before. Might be a cool way for me to learn. I'll take a shot at it this weekend.

**Comment by saurabhdaware** [comment]
Yeah perfect and no need to hurry you can take your time. Also I would suggest to start with some hello world project before directly working on this one. :smile:

**Comment by saurabhdaware** [comment]
Resolved in PR #5, Published in [v1.0.6](https://marketplace.visualstudio.com/items?itemName=saurabh.terminal-manager) 
Thanks @4a-42 for contributing. 
Closing this issue.
